packages/flutter/lib/src/services/binding.dart

```diff
@@ -60,15 +75,17 @@ mixin ServicesBinding on BindingBase {
       rawLicenses.complete(rootBundle.loadString('LICENSE', cache: false));
     });
     await rawLicenses.future;
-    final Completer<List<LicenseEntry>> parsedLicenses = Completer<List<LicenseEntry>>();
+    final Completer<List<LicenseEntry>> parsedLicenses =
+        Completer<List<LicenseEntry>>();
     Timer.run(() async {
-      parsedLicenses.complete(compute(_parseLicenses, await rawLicenses.future, debugLabel: 'parseLicenses'));
+      parsedLicenses.complete(compute(_parseLicenses, await rawLicenses.future,
+          debugLabel: 'parseLicenses'));
```

Comment:
looks like you accidentally ran some formatting software here which messed things up a bit

Reply:
Reverted.